### PR TITLE
Create `ls` chapter

### PR DESCRIPTION
**Pull request by jessicard**

### CODE REVIEW

**Comment by jessicard** [comment]
Maybe I should put an example of the output of this command?

**Comment by jessicard** [comment]
@zee @kmanion Ready for review!

File: en-US/src/ls.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# ls
+
+```
+ls
+```
+
+This command asks the computer to show you what's in the current directory; `ls` is short for "list". After you hit enter, you should see all the files and subdirectories inside your working directory.
```
Comment (zspencer) [inline en-US/src/ls.md]: `ls` is short for "list directory contents" which may be useful context or maybe not?

Comment (zspencer) [inline en-US/src/ls.md]: Also, we may want to introduce the concept of the `-F` flag, which adds a `/` to directories to make it more clear that's what they are.

File: en-US/src/ls.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+```
+$ ls
+a-file.txt   folder   markdown-file.md   ruby-file.rb
```
Comment (zspencer) [inline en-US/src/ls.md]: We should probably standardize on either folder or directory. Not sure which.

**Pull request closed without merging**